apm-agent-core/src/main/java/co/elastic/apm/agent/impl/async/SpanInScopeRunnableWrapper.java

```diff
@@ -53,34 +50,16 @@ public SpanInScopeRunnableWrapper wrap(Runnable delegate, AbstractSpan<?> span)
     // In this case, this class acts as the boundary of user and agent code so we have to do the tedious exception handling here
     @Override
     public void run() {
+        System.out.println("Wrapper run");
```

Comment:
leftover?

Reply:
🤦‍♂️ of course! 
Debugging async scenarios is a lot of fun 😅